Tasks/Maven/pmdformaven.ts

```diff
@@ -10,25 +10,27 @@ import trm = require('vsts-task-lib/toolrunner');
 
 import ar = require('./analysisresult');
 
+export const toolName:string = 'PMD';
+
 // Adds PMD goals, if selected by the user
 export function applyPmdArgs(mvnRun: trm.ToolRunner):void {
-    mvnRun.arg(['jxr:jxr', 'pmd:pmd']); // JXR source code cross-referencing is a required pre-goal for PMD
```

Comment:
Discovered that JXR source code cross-referencing is _not_ a required pre-goal, and will result in broken links unless the cross-reference files are uploaded as well.